CleanWeather/Models/APIResponses/CityForecastAPIResponse.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  CityForecastAPIResponse.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 18/12/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+struct CityForecastAPIResponse: Codable, Equatable {
+
+    let hourly: Hour
+
+    struct Hour: Codable, Equatable {
```

Comment:
Hour does not feel like a good name for a strict, because I am expecting it to be a number between 0 and 24